ci/deps/azure-35-compat.yaml

```diff
@@ -3,23 +3,23 @@ channels:
   - defaults
   - conda-forge
 dependencies:
-  - beautifulsoup4==4.4.1
-  - bottleneck=1.2.0
-  - cython=0.28.2
-  - hypothesis>=3.58.0
+  - beautifulsoup4=4.4.1
+  - bottleneck=1.2.1
   - jinja2=2.8
-  - numexpr=2.6.1
-  - numpy=1.12.0
+  - numexpr=2.6.2
+  - numpy=1.13.3
   - openpyxl=2.4.0
   - pytables=3.4.2
   - python-dateutil=2.5.0
-  - python=3.5*
+  - python=3.5.*
```

Comment:
I don't think this actually matters  but ok

Reply:
It does not, but I made it consistent across jobs.